picturae_importer.py

```diff
@@ -142,7 +146,85 @@ def assign_col_dtypes(self):
 
         self.record_full = self.record_full.replace({'True': True, 'False': False})
 
-        self.record_full = self.record_full.replace(['', None, 'nan', np.nan], pd.NA)
+        self.record_full = self.record_full.replace([None, 'nan', np.nan, '<NA>'], '')
+
+        # removing leading apostrophes from data columns
+        for col_name in list(["start", "end"]):
+            self.record_full[f"{col_name}_date"] = self.record_full[f"{col_name}_date"].str.lstrip("\'")
+
+
```

Comment:
creates a renamed copy of an image if belonging to duplicate sheet with multiple barcodes